src/main/java/com/commercetools/sync/products/helpers/ProductReferenceResolver.java

```diff
@@ -184,35 +183,30 @@ public ProductReferenceResolver(@Nonnull final ProductSyncOptions productSyncOpt
      * in the CTP project, the resultant draft would remain exactly the same as the passed product draft
      * (without reference resolution).
      *
-     * @param productDraft the product draft to resolve it's category references.
+     * @param draftBuilder the product draft to resolve it's category references.
```

Comment:
It's a private JavaDoc, but please update it according to the change anyways, since its there

Reply:
9dc2261e